packaging/app.organicmaps.desktop.metainfo.xml

```diff
@@ -111,6 +111,25 @@
   </screenshots>
 
   <releases>
+   <release version="2024.07.08-3" date="2024-07-08">
+     <description>
+       <p>Highlights:</p>
+         <ul>
+           <li>Better address search in the USA based on US Census TIGER data</li>
+           <li>Fixed rendering of texts in Indic, Arabic, Thai, Lao, Khmer, Myanmar, Tibetan, Hangul, Hebrew scripts</li>
+           <li>OSM data as of July 2, 2024</li>
+           <li>Display major roads earlier on the world overview map</li>
```

Comment:
Please check other PR for correct release notes.

Reply:
Done